### ISSUE TITLE
Discussion: Add openssl executable within flipper download

### ISSUE THREAD

**Issue by moaazassali** [facebook/flipper#5123]
**Suggestion**
I suggest that flipper should come with its own openssl.exe and other needed files to solve some issues related to the app if the user already has OpenSSL installed. My experience is with windows only, so I am not sure if it is similar in MacOS or Linux.

**Reason**
The issue is that if someone already has OpenSSL installed on windows, flipper will try and access `openssl.cnf` files from the directory `C:\Program Files\Common Files\ssl/openssl.cnf` as seen by a previous open issue here: https://github.com/facebook/flipper/issues/5069. Although it seems that the directory it tries to access might be different for different people as mentioned in a similar open issue here as well: https://github.com/facebook/flipper/issues/4334.

**Solution**
Many programs come with their own openssl executable like Git, VSCode, and others. Having flipper come with its own openssl executable should fix this issue for all users on Windows, so they don't have to resort to a 'hack' by creating this directory Flipper is looking for and copying their existing openssl.cnf file there.

**Comment by passy** [comment]
Yes, we'd love to make the setup easier and bundle something like this with Flipper. We currently don't have the resources for this, but if someone else would like to submit a PR, we will review that.
